ansible/roles/passenger/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
       dest: /etc/apache2/ssl/intermediate.crt
     sudo: true
 
-  - name: Create Rails Apache config (SSL Enabled)
```

Comment:
Oh, I see what you mean. This passenger role can be completely rails agnostic though. I put in the option to set the template src path which means we can template over any whatever-config for apache (in the intoxiclock case, a node config for apache). I'll reword that.